packages/onboarding/src/HotspotOnboardingUtil.ts

```diff
@@ -402,8 +328,8 @@ export const getAssertData = async ({
 
     const dcInDollars = dcNeeded.div(new BN(100000))
     const oraclePrice = await getOraclePriceFromSolana({ connection, cluster })
-    const hntNeeded = dcInDollars.div(oraclePrice)
-    hasSufficientHnt = balances.hnt.gte(hntNeeded)
+    const hntNeeded = dcInDollars.toNumber() / oraclePrice.toNumber()
+    hasSufficientHnt = balances.hnt.toNumber() > hntNeeded
```

Comment:
This will explode if for some reason they have more hnt than javadcript supports

Reply:
@ChewingGlass updated. Can you take another look?